desktop/libs/indexer/src/data/solrconfigs/solrcloud/conf/schema.xml

```diff
@@ -55,15 +55,15 @@
 
     <fieldtype name="binary" class="solr.BinaryField"/>
 
-    <fieldType name="pint" class="solr.IntField"/>
+    <fieldType name="pint" class="solr.TrieIntField"/>
```

Comment:
These are the changes we need, you have a lot of conflicts in the other files, do you think you could provide just the needed changes?